docs/modules/data-connections/pages/data-connections-configuration.adoc

```diff
@@ -381,6 +381,147 @@ OPTIONS (
+      .setName("my-remote-hazelcast")
+      .setType("Hz")
+      .setProperty(
+              "client_xml", <1>
+              "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + <2>
+              "<hazelcast-client xmlns=\"http://www.hazelcast.com/schema/client-config\"" +
+              "      xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"" +
+              "      xsi:schemaLocation=\"http://www.hazelcast.com/schema/client-config" +
+              "      http://www.hazelcast.com/schema/client-config/hazelcast-client-config-5.3.xsd\">" +
+              "    " +
+              "    <cluster-name>dev</cluster-name>" +
+              "    <network>" +
+              "        <cluster-members>" +
+              "            <address>172.17.0.2:5701</address>" +
+              "        </cluster-members>" +
+              "    </network>" +
+              "</hazelcast-client>")
+      .setShared(true)
+  );
+----
+<1> (Required) Exactly one of `client_xml`, `client_yml`, `client_xml_path`, `client_yml_path` must be specified.
+<2> Hazelcast client configuration to connect to a remote cluster. See xref:clients:java.adoc#configuring-java-client[Configuring Java Client].
+External file can be specified using `client_xml_path` or `client_yml_path` property instead of embedded configuration file.
+--
+SQL::
++
+--
+Data connections created in SQL behave differently to those defined in members' configuration files or in Java.
+
+- To retain SQL-defined data connections after a cluster restart, you must enable SQL catalog persistence. This feature is available in the Enterprise Edition.
+- You can create or drop a data connection using SQL commands. To update a data connection, you need to drop and then recreate it.
+
+NOTE: While Hazelcast data connection can be created using SQL command it cannot yet be used from SQL, because there
+is no SQL connector for remote Hazelcast available.
+
+[source,SQL]
+----
+CREATE DATA CONNECTION "my-hazelcast-cluster"
+TYPE Hz
+SHARED
+OPTIONS (
+    'client_xml'= <1>
+    '<?xml version="1.0" encoding="UTF-8"?> <2>
+     <hazelcast-client xmlns="http://www.hazelcast.com/schema/client-config"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
+         http://www.hazelcast.com/schema/client-config/hazelcast-client-config-5.3.xsd">
+
+       <cluster-name>dev</cluster-name>
+       <network>
+         <cluster-members>
+	   <address>172.17.0.2:5701</address>
+         </cluster-members>
+       </network>
+      </hazelcast-client>'
+);
+----
+<1> (Required) Exactly one of `client_xml`, `client_yml`, `client_xml_path`, `client_yml_path` must be specified.
+<2> Hazelcast client configuration to connect to a remote cluster. See xref:clients:java.adoc#configuring-java-client[Configuring Java Client].
+External file can be specified using `client_xml_path` or `client_yml_path` property instead of embedded configuration file.
```

Comment:
```suggestion
You can specify an external file with the `client_xml_path` or `client_yml_path` property instead of using an embedded configuration file.
```